rust/operator-binary/src/restart_controller/pod.rs

```diff
@@ -81,14 +81,22 @@ pub async fn start(client: &Client, watch_namespace: &WatchNamespace) {
 }
 
 async fn reconcile(pod: Arc<Pod>, ctx: Arc<Ctx>) -> Result<Action, Error> {
+    tracing::info!("Starting reconciliation ..");
     if pod.metadata.deletion_timestamp.is_some() {
         // Object is already being deleted, no point trying again
+        tracing::info!("Pod is already being deleted, taking no action!");
```

Comment:
I'd probably downgrade this into a `debug!`.

Reply:
In principle I'm happy to, my reasoning for this was, that I wanted to always show a result for a pod at the info level.
If we downgrade this to debug pods that are currently deleting will just silently exit reconciliation - creating a case where the person reading the logs needs to know how the operator works to fully understand the log.

Reply:
That's fair, I'm just worried about the other case of "hey why'd your operator report both statuses at the same time?". But we can always revisit and downgrade if it becomes too unreasonable.

Reply:
ok